cypress.config.js

```diff
@@ -0,0 +1,39 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
+  defaultCommandTimeout: 60000,
+  requestTimeout: 60000,
+  responseTimeout: 60000,
+  reporter: 'cypress-mochawesome-reporter',
```

Comment:
I'm not as familiar with this repo, but I see this was changed from `cypress-multi-reporters` to `cypress-mochawesome-reporter`.

Inside `reporter-config.json` I see:

```
{
  "reporterEnabled": "spec, mocha-junit-reporter",
  "mochaJunitReporterReporterOptions": {
    "mochaFile": "cypress/results/results-[hash].xml"
  }
}
```

Can the multi reports support the mochawesome you are adding here?

Reply:
We can check this part of course

Reply:
@jakubp-eliatra, WDYT?